azure-pipelines.yml

```diff
@@ -33,12 +33,12 @@ jobs:
         steps:
         - bash: |
             echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-${{ value }};google_apis;x86'
-            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' --force
+            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' -d 9 --force
 
             $ANDROID_HOME/emulator/emulator -list-avds
 
             echo "Starting emulator"
-            nohup $ANDROID_HOME/emulator/emulator -avd xamarin_android_emulator -no-snapshot > /dev/null 2>&1 &
+            nohup $ANDROID_HOME/emulator/emulator -avd xamarin_android_emulator -skin 1080x1920 -no-snapshot > /dev/null 2>&1 &
```

Comment:
We need different resolution for different devices. From 720x1280 to 1440x3040. You can see all possible resolutions in AVD manager. It is about a new task.

Reply:
opened an issue https://github.com/omen273/crosswordApp/issues/64